<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsignacionProf</class>
 <widget class="QWidget" name="AsignacionProf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asignacion de profesores</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/lentejas-con-chorizo-830.jpg</normaloff>../icons/lentejas-con-chorizo-830.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBox_2"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>                  -&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Asignar</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Regresar</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
